### DISCUSSION TITLE
FreeTube Release v0.14.0 Beta

### DISCUSSION THREAD

**Discussion by PrestonN** [FreeTubeApp/FreeTube#1649] [Announcements]
## Features
- **Most** settings (including locale changes) and history are now properly synchronized across windows (everything else is a WIP)
- The Invidious instance is now randomized on startup
  - The user can save the currently set Invidious instance as the default value for subsequent startups in the settings
- When a search filter is applied, the filter icon now has a colored ring around it to indicate that filters are used
- You can change the volume of the video by scrolling over the video player
- Pasting links to videos inside a playlist now opens the video in the playlist
- Captions settings (e.g. font size) are now persisted onto disk
- You can now open videos in an external Video player (Currently supported: MPV)
- Custom definable video forward and rewind intervals
- New Trending pages for music, gaming & movies
- You can navigate to subscriptions on FreeTube logo click
- Support for importing/exporting csv YouTube subscriptions

## Fixes
- Comments are now working again
- Importing subscriptions from NewPipe no longer fails if the file contains subscriptions from services other than YouTube
- When a video does not have any comments, FT will not create an error anymore
- Emojis in profile bubbles will no longer display a replacement character (�)
- When the player gets muted, the player volume's session value is now correctly set to 0
- The trending and most popular buttons are now properly hidden in portrait mode if set as such in the settings.
- If the autoplay countdown is in effect and the user navigates out of that page, the app will no longer display the remaining toasts related to the rest of the countdown
- Fixed a bug where disabling smooth scrolling didn't work unless the user manually closed and reopened the app
- Fixed a bug where, very occasionally, an error window related to `setNormalBounds` would pop up when closing 
- Window titles now respect your locale settings
- Live chat and comments now escape HTML characters
- Playlist description no longer overflows and hides the share playlist button
- YouTube Shorts are now accepted in the search bar
- Channel elements now are dynamic when the window is smaller
- Featured channel icons now display properly
- The description element does not show when the description is empty
- Playlists now play the next video when using Invidious API
- The media player does not show current time as tooltip when previewing over video
- Fixed buttons across app to toggle instead of them triggering the button again
- Large changelog messages now scroll

## Changes
- The https://yewtu.be Invidious instance has enabled their API for FreeTube and can now used. It will also be shown in the instance list within settings
- Enables the following languages: Estonian, Serbian
- Copying or opening links of videos in a playlist now contain the playlist information for YouTube and Invidious
- The large play button in the video player can be disabled in the settings, so screenshots can be taken more easily
- The spellchecker for the top search bar has been disabled
- The search bar's filters' menu appearance has been improved
- The playlist page now allows you to navigate to the author's channel; clicking the the playlist's thumbnail navigates you to the first video of the playlist
- The reload button for feeds is now fixed on scroll
- Theater mode button is now in the media player controls
- Improvement to speeds of reimporting subscriptions

**Comment by PrestonN** [comment]
Hey All!

As you can see, we have a very large list of patch notes for this release. Unfortunately one of the issues with the previous release is that the pop-up doesn't show all of the notes properly due to a fixed size, so we'll post it here instead so that the pop-up within the app is still accessible.

Do you like this approach? Should we use this more moving forward? Let us know!

**Reply by stephan-dev** [reply]
> Do you like this approach? Should we use this more moving forward? Let us know!

Thanks for fixing comments again, and some sync problems between windows. For notifications : it's nice to have both, a summary in the app, full details here.

**Comment by Tiltowait2** [comment]
THANK you for fixing comments.  That one was really hurting. :(  Good to see it back again!

**Comment by BaheySaied** [comment]
Hi :) .... Always Supporting You 😍

I Have Sent an E-mail to You For Only One thing ... Please Add " Egypt " In Trending Countries :) ... Please Please 😥

**Reply by peepo5** [reply]
Make an issue on our github issues tab.

**Comment by Button4Button** [comment]
Hi,
Can't thank you enough for fixing the comments... big thanks too for all you do for this project.
